typescript/rest-nextjs-api-routes-auth/prisma/seed.ts

```diff
@@ -1,4 +1,4 @@
-import { PrismaClient, Prisma } from '@prisma/client'
+import { PrismaClient } from '@prisma/client'
```

Comment:
Hi there, we do need that import for the type on line 5. 

```suggestion
import { PrismaClient } from '@prisma/client'
import type { Prisma } from '@prisma/client'
```

Reply:
Hmm I didn't get an error by removing it. Also, it wasn't imported as "type" before. Are both imports equal? (the one with type and the one without?)

Reply:
The tests on this PR are failing, unfortunately, so we can't remove it. 

They're not strictly equivalent but it works similar/the same